PerDiem/PerDiem/TransactionCell.m

```diff
@@ -39,32 +44,28 @@ - (void)setTransaction:(Transaction *)transaction {
     self.summaryLabel.text = self.transaction.summary;
     self.amountLabel.text = [self.amountFormatter stringFromNumber:self.transaction.amount];
 
-    if ([self.transaction.future boolValue]) {
-        [self styleFutureCell];
-    } else {
-        [self styleDefaultCell];
-    }
-    
-    self.backgroundColor = [UIColor transactionColor];
-//    self.backgroundView = [UIView new];
-//    self.selectedBackgroundView = [UIView new];
+//    if ([self.transaction.future boolValue]) {
```

Comment:
I left in in place in case we wanted to enable custom styling for future transactions, though it might require more refactoring so I may as well just remove it